runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/kotlinClasspathUtils.kt

```diff
@@ -39,6 +39,7 @@ private fun KotlinCompilation.compileClasspathOf(project: Project): FileCollecti
         return this.classpathOf(project)
     }
 
+    @Suppress("DEPRECATION")
     val platformDependencyFiles: FileCollection = (this as? AbstractKotlinNativeCompilation)
```

Comment:
Without context it's unclear which deprecated statement is being suppressed exactly, can it be extracted into a separate variable to reduce scope? Otherwise if something else from these chained calls gets deprecated - we won't notice it because of the suppression, I've already stumbled upon this very problem [when updating Dokka to 1.8.10](https://github.com/Kotlin/dokka/pull/2797/files#diff-b83ec3e7af23a13d4649e9bcbceb300867a0d602108281677b2c81c4170fb568).

And while I believe Anton, "promised" doesn't sound like a commitment :) Is it possible to create an issue in YouTrack, assight it to Anton and set the target version or something? To not end up in a situation where there's no replacement and the deprecated part has been deleted from KGP already

Reply:
I have created KT-58640